deploy/crds/keycloak.org_keycloaks_crd.yaml

```diff
@@ -31,13 +31,25 @@ spec:
         spec:
           description: KeycloakSpec defines the desired state of Keycloak.
           properties:
+            credentialSecret:
+              description: 'External only: The secret where the admin credentials
+                are to be found.'
+              type: string
+            endpoint:
+              description: 'External only: The URL to use for the keycloak admin API.'
+              type: string
             extensions:
               description: A list of extensions, where each one is a URL to a JAR
                 files that will be deployed in Keycloak.
               items:
                 type: string
               type: array
               x-kubernetes-list-type: set
+            external:
+              description: When set to true, this Keycloak will be marked external
+                and not be managed by this operator. It can then be used to create
+                realms, clients and users.
+              type: boolean
```

Comment:
Another option may be to put credentialSecret and endpoint as properties of external. Then having the properties at all implies external.

Reply:
Sure, that would work. I would then add a `external.enabled` property as we're doing in `externalAccess.enabled`? I guess I would then have the same syntax in the `KeycloakRealm` CRD.